Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/StartupHandler.java

```diff
@@ -26,10 +26,6 @@
 
   public static void HandleInit(FragmentActivity parent)
   {
-    // Ask the user to grant write permission if relevant and not already granted
-    if (DirectoryInitialization.isWaitingForWriteAccess(parent))
-      PermissionsHandler.requestWritePermission(parent);
```

Comment:
Because you removed this, now TvMainActivity is no longer calling `requestWritePermission`.

Reply:
Oh, I'll add that back then